libraries/chain/include/eosio/chain/wasm_interface.hpp

```diff
@@ -63,6 +63,9 @@ namespace eosio { namespace chain {
          //Immediately exits currently running wasm. UB is called when no wasm running
          void exit();
 
+         //Returns true if the code is cached
+         size_t is_code_cached(const digest_type& code_hash, const uint8_t& vm_type, const uint8_t& vm_version) const;
```

Comment:
I think you meant for the return value to be `bool`?

Reply:
Yes, thanks. Fixed.